.travis.yml

```diff
@@ -23,10 +22,8 @@ matrix:
           python: "2.6"
         - env: DJANGO='Django==1.8.*'
           python: "2.6"
-        - env: DJANGO='Django==1.3.*'
-          python: "3.4"
-        - env: DJANGO='Django==1.4.*'
-          python: "3.4"
+        - env: DJANGO='Django==1.9.*'
+          python: "2.6"
```

Comment:
Yes, you can see that python 2.6 is in the exclude section.
Travis only runs tests on Django 1.9 with python 2.7 and 3.4
https://travis-ci.org/disqus/django-mailviews/builds/104571281
Sorry, but i can't see the problem :wink: